include/oneapi/dpl/execution

```diff
@@ -18,11 +18,7 @@
 
 #include "oneapi/dpl/internal/common_config.h"
 // ATTENTION!!! Include the header from the C++ standard library before the oneDPL config header to see whether the _PSTL_VERSION macro is defined
-#if __cplusplus >= 201703L
-#    if __has_include(<execution>)
-#        include <execution>
-#    endif // __has_include(<execution>)
```

Comment:
I have reconsidered my earlier opinion regarding requiring the <execution> header to be present. Let's keep the checks for its presence, please.